pkg/cli/git/git.go

```diff
@@ -34,12 +36,20 @@ func Push(org, name, location, gitURL string, debug bool) error {
 	if err != nil {
 		return errors.Wrap(err, "Failed to parse git url")
 	}
-	// TODO username+password are created by server action before git push
-	// The values could be returned from some POST if we had some init command, but
-	// for now let's assume the values are fixed
 	username := config.DefaultUserName(org)
 	password := config.DefaultUserPassword
 
+	if uname != nil {
+		username = *uname
+	}
+	if pass != nil {
+		password = *pass
+	}
+	envPassword, exists := os.LookupEnv("GITEA_PROJECT_PASSWORD")
```

Comment:
nit: it should be `GIT_PROJECT_PASSWORD` or `FUSEML_PROJECT_PASSWORD` or even `FUSEML_PASSWORD`, because gitea is an implementation detail that may vary from codeset to codeset.

Reply:
Right. I thought you'll have some idea about this :-)